ir_x86.h

```diff
@@ -87,7 +87,11 @@ enum _ir_reg {
 #define IR_REG_GP_LAST  (IR_REG_FP_FIRST - 1)
 #define IR_REG_FP_LAST  (IR_REG_NUM - 1)
 
-#define IR_REGSET_64BIT 0
+#ifdef _M_X64 /* MSVC */
+# define IR_REGSET_64BIT 1
+#else
+# define IR_REGSET_64BIT 0
+#endif
```

Comment:
I think we don't need 64-bits for regset (32-bits should be enough). The number of CPU registers on Linux x86_64 and Windows 64 are the same.

Reply:
The reason to change this was other warnings like below, which signify issues with a regression potential:

`.\ir_ra.c(488): warning C4334: '<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)`

I'll dig more on this, check to revert and and perhaps try to fix these cases locally at where they're reported.

Thanks